README.md

```diff
@@ -24,8 +31,11 @@ What it does :
 
```

Comment:
Did you mean "Add plugin to your project"? The ordering of steps 1 and 2 need to be reversed in that case.